hub-js/src/local/resolver/field/spec-value-field.ts

```diff
@@ -87,13 +87,15 @@ async function resolveMutationReturnValue(
     newValue = resp[tiId];
   }
 
+  console.log(context.updateArgs.GetOwnerID(fetchInput.typeInstance.id));
```

Comment:
It would be good to remove it, or, alternatively, log with some context message, as now it might be confusing:

![Screenshot 2022-03-10 at 14 09 51](https://user-images.githubusercontent.com/7155799/157668413-e64b6a6b-20c4-42c2-ba0d-426e9de07978.png)

But I believe this is a leftover from implementing the changes 🙂